WLC/code_executor/haskell_executor.py

```diff
@@ -7,10 +13,25 @@ def __init__(self, ip="", port=""):
         super().__init__("haskell", PythonCodeFixer, ip, port)
 
     def execute_local(self, code):
-        raise NotImplemented()
+        file_code = tempfile.NamedTemporaryFile(delete=False, suffix='.hs')
```

Comment:
So the file will not be deleted when we are done with it?

Reply:
It will be by doing os.unlink. If we had delete=True it would get deleted right after we do .close() which is too early.